### PR TITLE
(qmlpref 4) correct main.qml with respect to prefs.

### PR DESCRIPTION
**Pull request by janiversen**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [x ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

in general change prefs.credentialStatus to PrefCloudStorage

Change a couple of credentialStatus checks in GlobalDrawer

Fix a missing parameter in saveCredentials()

### CODE REVIEW

File: mobile-widgets/qml/main.qml [hunk 1]
```diff
@@ -321,21 +321,12 @@ Kirigami.ApplicationWindow {
...
-					enabled: prefs.credentialStatus === CloudStatus.CS_VERIFIED ||
-							prefs.credentialStatus === CloudStatus.CS_NOCLOUD
+					enabled: PrefCloudStorage.cloud_verification_status === CloudStatus.CS_VERIFIED
 					onTriggered: {
-						if (prefs.credentialStatus === CloudStatus.CS_NOCLOUD) {
-							returnTopPage()
-							prefs.oldStatus = prefs.credentialStatus
-							manager.startPageText = "Enter valid cloud storage credentials"
-							prefs.credentialStatus = CloudStatus.CS_UNKNOWN
-							globalDrawer.close()
```
Comment (dirkhh) [inline mobile-widgets/qml/main.qml:332]: I think you are misunderstanding the intent. The logic was that if you hit "manually sync" and are in no-cloud mode, you are prompted to log in with a cloud account. So you are removing an intentional feature.
I'm not saying you are wrong to remove it, just that the way you describe this is a bit misleading.

**Review by dirkhh — APPROVED** [review]
While I disagree with the characterization in one of the commits, that's something I can address when merging by amending the commit message. I also wonder about the order of commits in the overall series, but not enough to make you re-push it :-)
I think this is pretty much ready to go in. I'll wait for the builds and then merge it manually.

File: mobile-widgets/qml/main.qml (hunk 1 above)
Comment (janiversen) [reply mobile-widgets/qml/main.qml:332]: Yes I really misunderstood that, and as a User I would be confused by getting the login screen without a warning or explanation.

Please feel free to edit the commit messages.

**Comment by dirkhh** [comment]
manually merged

**Pull request closed without merging**